app/src/main/java/com/duckduckgo/app/browser/BrowserActivity.kt

```diff
@@ -336,6 +336,15 @@ open class BrowserActivity : DuckDuckGoActivity() {
                 Timber.d("Favorite clicked from widget $sharedText")
                 lifecycleScope.launch { viewModel.onOpenFavoriteFromWidget(query = sharedText) }
                 return
+            } else if (intent.getBooleanExtra(OPEN_IN_CURRENT_TAB_EXTRA, false)) {
```

Comment:
And also because tapping on a Favourite needs to open the current tab, and not create a new one like it did before. This will also be used from Shortcuts, when a link to a url will open in the current tab, and not a new one.